test/docfx.Test/lib/ParallelUtilityTest.cs

```diff
@@ -20,7 +20,12 @@ public static async Task ThrowsTheSameException(Type exceptionType)
             var exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), Run));
             Assert.Equal(exceptionType, exception.GetType());
 
+            exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), RunExpandChild));
```

Comment:
Shall we put this in a separate test? If the test fails It would be hard to see which case is failing

Reply:
then all the `[TheoryData]` would be duplicated

Reply:
yes, the duplication of `[TheoryData]` should be fine